lib/ansible/modules/pip.py

```diff
@@ -611,6 +612,60 @@ def setup_virtualenv(module, env, chdir, out, err):
     return out, err, cmd
 
 
+def _resolve_package_names(
+        module: AnsibleModule,
+        package_list: list[Package],
+        pip: list[str],
+        python_bin: str
```

Comment:
nit: trailing commas help with better diffs.
```suggestion
        python_bin: str,
```

Reply:
It looks like this change was overlooked.